### ISSUE TITLE
gconv modules not available in container runtime

### ISSUE THREAD

**Issue by smcv** [ValveSoftware/steam-runtime#242]
Reported by @XakepSDK in #234. I'm splitting that issue report up because it seems to be two separate issues that affect the same game.

#### Your system information

* Steam Runtime Version: 0.20200204.0
* Distribution (e.g. Ubuntu 18.04): Arch
* Link to your full system information (Help -> System Information) in a [Gist](https://gist.github.com/): https://gist.github.com/XakepSDK/4980d0a277c12015bb5d1b81f8501c6f
* Have you checked for system updates?: yes
* Are you using the Steam Linux Runtime compatibility tool?: yes

#### Please describe your issue in as much detail as possible:

> When i start game with SLR, game has no text. Instead of actual text, i see text keys, e.g. UI_menu_exit

In the game's terminal output, we see:

```
[0][00:00:31.112][ERR] There was an error converting a localization file from UTF-16LE to UCS-4LE.
[0][00:00:31.112][ERR] be sure output from 'iconv -l' contains those two encodings.
[0][00:00:31.112][ERR] If it does, then ensure you have 32bit iconv libs located in /usr/lib/gconv.
[0][00:00:31.112][ERR] If you don't have them try to install a package with 'glibc locale' libs in it.
```

#### Steps for reproducing this issue:

Run Shadow Warrior (ID 233130) with pressure-vessel container runtime (SteamLinuxRuntime), on a non-Debian-derived host system. Observe the main menu.

**Comment by smcv** [comment]
I suspect this is happening because pressure-vessel has imported the host system's glibc into the container (we have to do this to make the host system's graphics drivers work) but hasn't also made the gconv modules available. The host system's glibc doesn't find the older gconv modules from the Steam Runtime either, because they're in `/usr/lib/*-linux-gnu` (Debian-style multiarch) and Arch systems aren't configured for multiarch.

**Comment by XakepSDK** [comment]
`iconv -l` output in scout env, if it may help
https://gist.github.com/XakepSDK/7db63683e3ff9230a25729560be18976

**Comment by smcv** [comment]
That's consistent with what I thought. I imagine those are the ones that are directly implemented in your glibc rather than needing a plugin - they seem like a very basic set (ASCII, UCS-2, UCS-4 and UTF-8). The game also wants UTF-16, which isn't in that set.

This should be fixable, we have similar special cases for things like locales already.

**Comment by smcv** [comment]
This didn't make it into the 26 March Steam client update, but will hopefully be fixed in the next Steam Runtime update after that.

**Comment by XakepSDK** [comment]
Fixed in SLR 0.20200415.0

New issue: https://github.com/ValveSoftware/steam-runtime/issues/254

**Comment by kisak-valve** [comment]
Closing per the last comment.
